src/psycopmlutils/timeseriesflattener/flattened_dataset.py

```diff
@@ -73,18 +76,19 @@ def __init__(
         # in self.df
         self.df_aggregating = self.pred_times_with_uuid
         self.df = self.pred_times_with_uuid
+        self.loaders_catalogue = data_loaders
 
-    def add_predictors_from_list_of_argument_dictionaries(
+    def add_temporal_predictors_from_list_of_argument_dictionaries(
         self,
         predictor_list: List[Dict[str, str]],
-        predictor_dfs_dict: Dict[str, DataFrame],
+        predictor_dfs_dict: Dict[str, DataFrame] = None,
         resolve_multiple_fn_dict: Optional[Dict[str, Callable]] = None,
     ):
         """Add predictors to the flattened dataframe from a list.
 
         Args:
             predictor_list (List[Dict[str, str]]): A list of dictionaries describing the prediction_features you'd like to generate.
```

Comment:
remove type hint for argument name
```suggestion
            predictors (List[Dict[str, str]]): A list of dictionaries describing the prediction_features you'd like to generate.
```